addons/binding/org.openhab.binding.gpstracker/ESH-INF/thing/tracker.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="gpstracker"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<!-- Thing Types -->
+	<thing-type id="tracker">
+		<label>Tracker Device</label>
+		<description>Device running tracker application</description>
+		<channels>
+			<channel id="lastReport" typeId="lastReport"/>
+			<channel id="location" typeId="location"/>
```

Comment:
Please use the system.location channel type instead of defining your own - see https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels